src/Components/Milestone.css

```diff
@@ -14,11 +14,23 @@
   line-height: 1;
 }
 
+.p-card {
+  border-radius: 1rem;
+}
```

Comment:
We are using PrimeReact, this can be achieved without the extra CSS https://www.primefaces.org/primeflex/borderradius

Reply:
Sure eddie will do it

Reply:
Great 👍 more open source green squares for you 🎉